hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -179,7 +179,7 @@ def children(self):
 
 class ExampleProperty(object):
     """There are many properties of examples that we calculate by
-    essentially rerunning thet test case multiple times based on the
+    essentially rerunning that test case multiple times based on the
```

Comment:
I think this is supposed to be `the` here.

Reply:
Fixed